--- a/opencog/scm/opencog.scm
+++ b/opencog/scm/opencog.scm
@@ -29,15 +29,15 @@
 ;
 
 ; lib64 is used by various versions of CentOS
-(define path "/usr/local/lib/opencog:/usr/local/lib64/opencog")
+(define path "/usr/lib/opencog:/usr/lib64/opencog")
 (setenv "LTDL_LIBRARY_PATH"
 	(if (getenv "LTDL_LIBRARY_PATH")
 		(string-append (getenv "LTDL_LIBRARY_PATH") ":" path)
 		path))
 
 ; Work-around another common usability issue...
-(add-to-load-path "/usr/local/share")
-(add-to-load-path "/usr/local/share/opencog/scm")
+(add-to-load-path "/usr/share/guile/site/2.2/")
+(add-to-load-path "/usr/share/guile/site/2.2/opencog")
 
 (define-module (opencog))
 (load-extension "libsmob" "opencog_guile_init")
