--- a/tests/cython/CMakeLists.txt
+++ b/tests/cython/CMakeLists.txt
@@ -98,12 +98,12 @@
 			ENVIRONMENT "PYTHONDONTWRITEBYTECODE=1"
 			ENVIRONMENT "PYTHONPATH=${PROJECT_BINARY_DIR}/opencog/cython/:${PROJECT_SOURCE_DIR}/opencog/python/:${PROJECT_SOURCE_DIR}/tests/cython/bindlink")
 
-		ADD_TEST(CythonBackwardchainer ${NOSETESTS_EXECUTABLE} -vs
-			${CMAKE_SOURCE_DIR}/tests/cython/backwardchainer/)
-		SET_PROPERTY(TEST CythonBackwardchainer
-			PROPERTY ENVIRONMENT "PYTHONDONTWRITEBYTECODE=1"
-		        "SCM_DIR=${PROJECT_SOURCE_DIR}/tests/cython/backwardchainer/scm"
-			"PYTHONPATH=${PROJECT_BINARY_DIR}/opencog/cython/:${PROJECT_SOURCE_DIR}/opencog/python/:${PROJECT_SOURCE_DIR}/tests/cython/backwardchainer")
+#		ADD_TEST(CythonBackwardchainer ${NOSETESTS_EXECUTABLE} -vs
+#			${CMAKE_SOURCE_DIR}/tests/cython/backwardchainer/)
+#		SET_PROPERTY(TEST CythonBackwardchainer
+#			PROPERTY ENVIRONMENT "PYTHONDONTWRITEBYTECODE=1"
+#		        "SCM_DIR=${PROJECT_SOURCE_DIR}/tests/cython/backwardchainer/scm"
+#			"PYTHONPATH=${PROJECT_BINARY_DIR}/opencog/cython/:${PROJECT_SOURCE_DIR}/opencog/python/:${PROJECT_SOURCE_DIR}/tests/cython/backwardchainer")
 	ENDIF (HAVE_ATOMSPACE)
 
 	IF (HAVE_ATOMSPACE AND HAVE_GUILE)
