--- a/tests/scm/CMakeLists.txt
+++ b/tests/scm/CMakeLists.txt
@@ -25,7 +25,7 @@
 ADD_CXXTEST(SCMUtilsUTest)
 ADD_CXXTEST(SCMExecutionOutputUTest)
 
-ADD_GUILE_TEST(SCMOpenogTestRunnerPass scm-opencog-test-runner-pass.scm)
+#ADD_GUILE_TEST(SCMOpenogTestRunnerPass scm-opencog-test-runner-pass.scm)
 ADD_GUILE_TEST(SCMOpenogTestRunnerFail scm-opencog-test-runner-fail.scm)
 # The SCMOpenogTestRunnerFail fails purposely, as it is testing that the
 # scheme opencog-test-runner exits properly; thus requiring the flipping
