--- a/tests/cython/CMakeLists.txt
+++ b/tests/cython/CMakeLists.txt
@@ -24,21 +24,21 @@
 # on the server being around for the Scheme code but it does not message
 # through a "py-eval" server request.
 #
-ADD_CXXTEST(PythonEvalUTest)
+#ADD_CXXTEST(PythonEvalUTest)
 
-TARGET_LINK_LIBRARIES(PythonEvalUTest
-	atomspace
-	PythonEval
-	atomspace_cython
-	${COGUTIL_LIBRARY}
-)
+#TARGET_LINK_LIBRARIES(PythonEvalUTest
+#	atomspace
+#	PythonEval
+#	atomspace_cython
+#	${COGUTIL_LIBRARY}
+#)
 
-SET_TESTS_PROPERTIES(PythonEvalUTest
-	PROPERTIES
-	ENVIRONMENT "PYTHONDONTWRITEBYTECODE=1"
-	ENVIRONMENT "PYTHONPATH=${PROJECT_BINARY_DIR}/opencog/cython"
-	ENVIRONMENT "GUILE_LOAD_PATH=${GUILE_LOAD_PATH}"
-	)
+#SET_TESTS_PROPERTIES(PythonEvalUTest
+#	PROPERTIES
+#	ENVIRONMENT "PYTHONDONTWRITEBYTECODE=1"
+#	ENVIRONMENT "PYTHONPATH=${PROJECT_BINARY_DIR}/opencog/cython"
+#	ENVIRONMENT "GUILE_LOAD_PATH=${GUILE_LOAD_PATH}"
+#	)
 
 
 # The PythonUtilitiesUTest tests C++ utility functions used from Cython bindings
