--- a/tests/cython/CMakeLists.txt
+++ b/tests/cython/CMakeLists.txt
@@ -44,19 +44,19 @@
 # The PythonUtilitiesUTest tests C++ utility functions used from Cython bindings
 # outside of those bindings so failures are easier to debug.
 #
-ADD_CXXTEST(PythonUtilitiesUTest)
+#ADD_CXXTEST(PythonUtilitiesUTest)
 
-TARGET_LINK_LIBRARIES(PythonUtilitiesUTest
-	PythonEval
-	utilities_cython
-	atomspace_cython
-	atomspace
-	${COGUTIL_LIBRARY}
-)
-SET_TESTS_PROPERTIES(PythonUtilitiesUTest
-	PROPERTIES
-	ENVIRONMENT "PYTHONDONTWRITEBYTECODE=1"
-	ENVIRONMENT "PYTHONPATH=${PROJECT_BINARY_DIR}/opencog/cython")
+#TARGET_LINK_LIBRARIES(PythonUtilitiesUTest
+#	PythonEval
+#	utilities_cython
+#	atomspace_cython
+#	atomspace
+#	${COGUTIL_LIBRARY}
+#)
+#SET_TESTS_PROPERTIES(PythonUtilitiesUTest
+#	PROPERTIES
+#	ENVIRONMENT "PYTHONDONTWRITEBYTECODE=1"
+#	ENVIRONMENT "PYTHONPATH=${PROJECT_BINARY_DIR}/opencog/cython")
 
 
 IF (HAVE_NOSETESTS)
