--- a/tests/atoms/CMakeLists.txt
+++ b/tests/atoms/CMakeLists.txt
@@ -53,8 +53,8 @@
 ADD_CXXTEST(ValueOfUTest)
 TARGET_LINK_LIBRARIES(ValueOfUTest clearbox execution atomspace)
 
-ADD_CXXTEST(StreamUTest)
-TARGET_LINK_LIBRARIES(StreamUTest clearbox execution smob atomspace)
+#ADD_CXXTEST(StreamUTest)
+#TARGET_LINK_LIBRARIES(StreamUTest clearbox execution smob atomspace)
 
 ADD_CXXTEST(IdenticalLinkUTest)
 TARGET_LINK_LIBRARIES(IdenticalLinkUTest execution atomspace)
