--- a/opencog/util/backtrace-symbols.c
+++ b/opencog/util/backtrace-symbols.c
@@ -121,14 +121,14 @@
       if (spot->found)
             return;
 
-      if ((bfd_get_section_flags(abfd, section) & SEC_ALLOC) == 0)
+      if ((bfd_section_flags(section) & SEC_ALLOC) == 0)
             return;
 
-      vma = bfd_get_section_vma(abfd, section);
+      vma = bfd_section_vma(section);
       if (spot->pc < vma)
             return;
 
-      size = bfd_section_size(abfd, section);
+      size = bfd_section_size(section);
       if (spot->pc >= vma + size)
             return;
 
