--- a/tests/moses/CMakeLists.txt
+++ b/tests/moses/CMakeLists.txt
@@ -102,13 +102,13 @@
 	${COGUTIL_LIBRARY}
 )
 
-ADD_CXXTEST(dynFeatSelUTest)
-TARGET_LINK_LIBRARIES(dynFeatSelUTest
-	moses_exec
-	moses
-	comboreduct
-	${COGUTIL_LIBRARY}
-)
+#ADD_CXXTEST(dynFeatSelUTest)
+#TARGET_LINK_LIBRARIES(dynFeatSelUTest
+#	moses_exec
+#	moses
+#	comboreduct
+#	${COGUTIL_LIBRARY}
+#)
 
 ADD_CXXTEST(AntUTest)
 TARGET_LINK_LIBRARIES(AntUTest
