--- a/batch-process.sh
+++ b/batch-process.sh
@@ -20,9 +20,8 @@
 VM_OPTS="-Xmx1024m"
 
 RELEX_OPTS="\
-	-Djava.library.path=/usr/lib:/usr/lib/jni:/usr/local/lib:/usr/local/lib/jni \
-	-Drelex.algpath=data/relex-semantic.algs \
-	-Dwordnet.configfile=data/wordnet/file_properties.xml \
+	-Drelex.algpath=/usr/share/relex/data/relex-semantic.algs \
+	-Dwordnet.configfile=/usr/share/relex/data/wordnet/file_properties.xml \
 	"
 
 
