--- a/relation-extractor.sh
+++ b/relation-extractor.sh
@@ -29,7 +29,7 @@
 # VM_OPTS="-Xdebug -Xnoagent -Djava.compiler=none -Xrunjdwp:transport=dt_socket,server=y,suspend=y -Xmx1024m -Djava.library.path=/usr/lib:/usr/local/lib"
 
 VM_OPTS="-Xmx1024m \
-	-Djava.library.path=/usr/lib:/usr/lib/jni:/usr/local/lib:/usr/local/lib/jni"
+        -Dwordnet.configfile=/usr/share/relex/data/wordnet/file_properties.xml"
 
 # By default, these resources are read from the relex jar file.
 # Alternately, they are taken from the default paths, which are the
